packages/notification-service/app.integration.yaml

```diff
@@ -1,6 +1,6 @@
-runtime: nodejs
-env: flex 
-service: integration 
-instance_class: B4
-manual_scaling:
-  instances: 1
\ No newline at end of file
+runtime: nodejs8
+service: integration
+instance_class: F4
+automatic_scaling:
```

Comment:
I tried this when I first created the service and found that automatic scaling would always eventually kill the service since it wasn't receiving any inbound network requests. Seems like app engine is kind of hard wired to run web servers and not well suited to other kinds of apps...
It's possible this issue has since been fixed but please test that before changing this. 
Also worth it to double check that we have alerting for when the service goes down

Reply:
It seems like the version that I deployed has been running for a while - do you remember when it would get killed? https://console.cloud.google.com/appengine/instances?project=celo-org-mobile&instancesTablesize=20&versionId=20190926t191656

It also looks like we have an uptime check https://app.google.stackdriver.com/uptime/efe3d6cc3966ae961e398dd5b73d2094?project=celo-testnet

Does this seem okay to you then?

Reply:
Why use automatic scaling if min and max instances are both set to 1?